MediaGalleryUi/Ui/Component/Listing/Columns/Pages/Options.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryUi\Ui\Component\Listing\Columns\Pages;
+
+use Magento\Cms\Api\PageRepositoryInterface;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\Api\FilterBuilder;
+use Magento\Framework\Data\OptionSourceInterface;
+use Magento\Framework\Exception\LocalizedException;
+
+
+/**
+ * Image source filter options
+ */
+class Options implements OptionSourceInterface
+{
+    protected $_pageRepository;
+
+    protected $_searchCriteriaBuilder;
+
+    protected $_filterBuilder;
+
```

Comment:
Please, change `protected` to `private` and remove the underscore symbol. Add the property type in docblock.